libbeat/processors/add_cloud_metadata/_meta/fields.yml

```diff
@@ -44,3 +49,14 @@
       type: alias
       path: cloud.region
       migration: true
+
+    - name: meta.cloud.account.id
```

Comment:
These new alias are not needed. They were initially created for backwards compatibility with previous names. As these fields are new, we don't need to have them in the old layout

Reply:
I thought data from the processor was created under `meta.cloud ...` and then had to be aliased to ECS `cloud. ...`

idk how this works, but sounds like I just need to remove this block from fields.yml, and that _somehow_ the field will be mapped to ECS